test/test_runestone_components.py

```diff
@@ -111,6 +112,11 @@ def selenium_utils_user_ac(selenium_utils_user):
     return selenium_utils_user
 
 
+@pytest.mark.asyncio
```

Comment:
This doesn't need to be async, since it's a sync test.

Reply:
OK, I wasn't sure if the tests were being run in such a way that we had to define them all as async.